dashboard/src/actions/datasetListActions.js

```diff
@@ -4,20 +4,21 @@ import * as TYPES from "./types";
 import { DANGER, ERROR_MSG } from "assets/constants/toastConstants";
 
 import API from "../utils/axiosInstance";
+import Cookies from "js-cookie";
 import { showToast } from "./toastActions";
 import { uriTemplate } from "utils/helper";
 
 export const fetchPublicDatasets = (page) => async (dispatch, getState) => {
   try {
     dispatch({ type: TYPES.LOADING });
     const endpoints = getState().apiEndpoint.endpoints;
+    const loggedIn = Cookies.get("isLoggedIn");
```

Comment:
I noticed in the sidebar PR that you're using `keycloak.authenticated` apparently instead of this `Cookies.get("isLoggedIn")`. We should be consistent, I think: but I'm curious what the difference actually is, and if one might avoid the timing problem we've seen with transient authentication errors??

Reply:
Cookie `isLoggedIn` is set once `keycloak.authenticated` is true so no much difference in the timing.
Both are same, but to main consistency will update in the next PR.